packages/fake-data/src/fake/fake.ts

```diff
@@ -409,6 +409,7 @@ const MODEL_SPECIFIC_OVERRIDES = {
     const frequency = chance.pickone(REPEAT_FREQUENCY_VALUES);
     const endsMode = chance.pickone(['on', 'after']);
     return {
+      frequency,
```

Comment:
I think the issue is that depending on the frequency we need to define `nthWeekday`. We do that below however we weren't setting the frequency explicitly so there's a chance the two would be compatible.